packages/core/src/networks.ts

```diff
@@ -436,3 +436,44 @@ export const toSupportedChainId = (chainId: number): SupportedChainId => {
   }
   return chainId as SupportedChainId
 }
+
+/**
+ * The number of blocks that Hardhat rewinds when forking the given network. Rewinding the block
+ * number protects against chain reorgs. Copied from Hardhat:
+ * https://github.com/NomicFoundation/hardhat/blob/caa504fe0e53c183578f42d66f4740b8ec147051/packages/hardhat-core/src/internal/hardhat-network/provider/utils/reorgs-protection.ts
+ */
+export const getLargestPossibleReorg = (chainIdStr: string): bigint => {
```

Comment:
Why isn't this imported from Hardhat since we have that installed as a production dependency anyway?

Reply:
It's an "internal" directory, so I don't feel confident that the file location and the function signature will be the same in future minor/patch versions of Hardhat